### PR TITLE
export offset values and fix name

### PR DESCRIPTION
**Pull request by jsaltermedialab**

### CODE REVIEW

File: xml.go [hunk 1]
```diff
@@ -1,59 +1,87 @@
...
 type AdBreak struct {
+	XMLName        xml.Name       `xml:"vmap:AdBreak"`
 	TimeOffset     *Offset        `xml:"timeOffset,attr"`
 	BreakType      string         `xml:"breakType,attr"`
 	BreakID        string         `xml:"breakId,attr"`
 	RepeatAfter    string         `xml:"repeatAfter,attr"`
-	AdSource       AdSource       `xml:"AdSource"`
-	TrackingEvents TrackingEvents `xml:"TrackingEvents"`
-	Extensions     Extensions     `xml:"Extensions"`
+	AdSource       AdSource       `xml:"vmap:AdSource"`
```
Comment (cristianchaparroa) [inline xml.go:31]: why is aded the extra "vamp" tag?

**Review by cristianchaparroa — APPROVED** [review]

File: offset.go [hunk 2]
```diff
@@ -31,15 +31,19 @@ const (
 
 // MarshalText implements the encoding.TextMarshaler interface.
 func (off *Offset) MarshalText() ([]byte, error) {
-	if off.position == OffsetPositionStart || off.position == OffsetPositionEnd {
-		return []byte(off.position), nil
+	if off == nil {
```
Comment (kmui18) [inline offset.go:34]: Is this possible? This function has to be called on the object?

Comment (jsaltermedialab) [reply offset.go:34]: yeah its pointer. the xml package will basically call `nil.MarshalText()` on any Offset object regardless of if its a real object or not when its marshalling a vmap object

**Review by kmui18 — APPROVED** [review]

File: xml.go (hunk 1 above)
Comment (jsaltermedialab) [reply xml.go:31]: its an xml namespace so that the generated XML will pass validation

**Pull request merged**